app/styles/ui/_blank-slate.scss

```diff
@@ -57,5 +57,6 @@
 
 .blankslate-image {
   margin: var(--spacing-double) auto;
-  width: 400px;
+  min-width: 400px;
```

Comment:
Hey @shimpossible, sorry for taking so long on this. Could you please add a `max-width: 800px` here as well so we have some constraints on it?